JMdictFiles/parsing_JMDict.py

```diff
@@ -0,0 +1,29 @@
+import xml.etree.ElementTree as ET
```

Comment:
Please add module docstring in the first line of the file, explaining the need and usage of this script.